client/web/src/search/input/toggles/Toggles.tsx

```diff
@@ -102,7 +111,14 @@ export const Toggles: React.FunctionComponent<TogglesProps> = (props: TogglesPro
         submitOnToggle({ newPatternType })
     }, [patternType, setPatternType, settingsCascade.final, submitOnToggle])
 
-    const fullQuery = [navbarSearchQuery, `patternType:${patternType}`, caseSensitive ? 'case:yes' : '']
+    const fullQuery = [
+        showSearchContext && selectedSearchContextSpec && !isContextFilterInQuery(navbarSearchQuery)
+            ? `context:${selectedSearchContextSpec}`
+            : '',
```

Comment:
Will this add a leading space to the query? If it does, *maybe* something like this will work?

```suggestion
            : ...[],
```

Reply:
Same comment below for caseSensitive

Reply:
Nope, because we filter out the empty strings below with `.filter(queryPart => !!queryPart)`

Reply:
Ahh missed that, thanks!